index.bs

```diff
@@ -889,6 +792,102 @@ user consent to a specific transaction. The structure of these signatures is def
     [[#op-get-assertion]].
 </div>
 
+## Additional options for Attestation (dictionary <dfn dictionary>MakeCredentialOptions</dfn>) ## {#dictionary-makecredentialoptions}
```

Comment:
Editorial: this isn't "additional options" any more, it's all the options. Maybe say something like "Parameters for credential creation" instead?